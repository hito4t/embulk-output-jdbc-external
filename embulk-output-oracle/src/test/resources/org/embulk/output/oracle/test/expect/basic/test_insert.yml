table: TEST1
mode: insert
