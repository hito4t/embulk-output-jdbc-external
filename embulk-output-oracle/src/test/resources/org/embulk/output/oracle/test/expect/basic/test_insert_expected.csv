9999,,,,,,
A001,ABCDE,abcde,0,123.45,15-03-04,15-03-05 03:34:56.000000000
A002,ＡＢ,ａｂｃｄｅｆ,-9999,-100000000,15-03-05,15-03-06 14:59:59.000000000
A003,,,,,,
